tooling/github-actions/install/action.yml

```diff
@@ -0,0 +1,37 @@
+name: "Install"
```

Comment:
@michaelrambeau 

Note this file to setup the env and install pnpm.

I want to go back and change the other CI workflows to use this if you agree.

Reply:
If this new file is used only by GitHub actions, would it be possible to move it to `/.github` folder? (as I don't think it can re-used by anything else)

Also previously, PNPM was installed by using an action

``` yaml
      - name: Install PNPM 8
        uses: pnpm/action-setup@v2
        with:
          version: 8.9.0
```

I wonder why it's not possible to re-use this instead of installing PNPM by ourselves? (and avoid what you did with the PNPM cache, making the setup simpler and robust, maybe?)

Reply:
I liked this approach to keeping "utility" actions separate from `workflows/` as it allows us to keep each file in workflow as its own contained program as well as simplifying each workflow a little more.

But you're right, nothing is sharing yet.